### ISSUE TITLE
[RELEASE] Release version 1.3.1 Opensearch-Dashboards

### ISSUE THREAD

**Issue by abhinavGupta16** [opensearch-project/opensearch-build#1885]
## Release OpenSearch Dashboards 1.3.1

<details><summary>How to use this issue</summary>
<p>

## This Release Issue

This issue captures the state of the OpenSearch release, its assignee is responsible for driving the release. Please contact them or @mention them on this issue for help. There are linked issues on components of the release where individual components can be tracked.  More details are included in the Maintainers [Release owner](https://github.com/opensearch-project/opensearch-build/blob/main/MAINTAINERS.md#release-owner) section.

## Release Steps

There are several steps to the release process, these steps are completed as the whole release and components that are behind present risk to the release.  The release owner completes the tasks in this ticket, whereas component owners resolve tasks on their ticket in their repositories.

Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.

### Component List

To aid in understanding the state of the release there is a table with status indicating each component state. This is updated based on the status of the component issues.

</p>
</details>

### Preparation

- [x] Assign this issue to a release owner.
- [x] Declare a pencils down date for new features to be merged. 
- [x] __REPLACE_RELEASE-minus-14-days__ is pencils down date for feature freeze.
- [ ] Update the Campaigns section to include monitoring campaigns during this release. 
- [x] Update this issue so all `__REPLACE_RELEASE-__` placeholders have actual dates.
- [x] Document any new quality requirements or changes.
- [x] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1). 
- [ ] [Create a release issue in every component repo](https://github.com/opensearch-project/opensearch-build/blob/main/meta/README.md#create-a-release-issue) that links back to this issue, update Components section with these links.
- [ ] Ensure the label is created in each component repo for this new version, and the next minor release. [Create a version label](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#create-or-update-labels-in-all-plugin-repos)
- [ ] Ensure that all release issues created above are assigned to an owner in the component team.

### CI/CD (Feature Freeze) - April 1 2022

- [x] Create Jenkins workflows that run daily snapshot builds for OpenSearch and OpenSearch Dashboards. 
- [x] Increment each component version to 1.3.1 and ensure working CI in component repositories.
> - [x] https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/161
> - [x] https://github.com/opensearch-project/sql/issues/532
> - [x] https://github.com/opensearch-project/observability/issues/621
> - [x] Security-Dashboards
> - [x] https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/issues/227
> - [x] https://github.com/opensearch-project/dashboards-reports/issues/330
> - [x] https://github.com/opensearch-project/dashboards-visualizations/issues/60
> - [x] https://github.com/opensearch-project/index-management-dashboards-plugin/issues/170
> - [x] https://github.com/opensearch-project/alerting-dashboards-plugin/issues/206 

- [x] Make pull requests to add each component to [manifests/1.3.1/opensearch-dashboards-1.3.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/1.3.1/opensearch-dashboards-1.3.1.yml) with the corresponding checks.
- [x] OpenSearch-Dashboards core and components teams finalize their features

### Campaigns

__REPLACE with OpenSearch wide initiatives to improve quality and consistency.__

### Code Complete - April 1 2022

- [x] Code Complete: Make sure that the code for this specific version of the release is ready and the branch corresponding to this release has been added to this release version manifest.
- [x] Verify pull requests to add each component to [manifests/1.3.1/opensearch-dashboards-1.3.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/1.3.1/opensearch-dashboards-1.3.1.yml) have been merged.
- [ ] Gather, review and combine the release notes from components repositories.

### Release testing - April 4 2022

- [x] Declare a release candidate build, and provide the instructions with the release candidates for teams on testing (__REPLACE_RELEASE-minus-8-days__).
- [ ] Sanity Testing (__REPLACE_RELEASE-minus-8-days__ - __REPLACE_RELEASE-minus-6-days__): Sanity testing and fixing of critical issues found by teams. Teams test their components within the distribution, ensuring integration, backwards compatibility, and perf tests pass.
- [x] Publish all test results in the comments of this issue.

### Performance testing validation - _Ends __REPLACE_RELEASE-minus-6-days___

- [ ] ~~Performance tests do not show a regression~~
- [ ] ~~Longevity tests do not show any issues~~

### Release - April 7 2022

- [ ] Verify [all issues labeled `v1.3.1` in all projects](https://github.com/opensearch-project/project-meta#find-labeled-issues) have been resolved.
- [x] Complete [documentation](https://github.com/opensearch-project/documentation-website) for this release.
- [ ] Author [blog post](https://github.com/opensearch-project/project-website) for this release.
- [x] __REPLACE_RELEASE-minus-1-day - Publish this release on [opensearch.org](https://opensearch.org/downloads.html).
- [ ] __REPLACE_RELEASE-day - Publish a [blog post](https://github.com/opensearch-project/project-website) - release is launched!

### Post Release

- [x] Create [release tags](https://github.com/opensearch-project/opensearch-build/issues/378#issuecomment-999700848) for each component.
- [x] Replace refs in [manifests/1.3.1](/opensearch-project/opensearch-build/tree/main/manifests/1.3.1) with tags and remove checks.
- [x] Generate distribution release notes reviewed by PM team for opensearch-build repository.
- [x] Prepare [for next patch release](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#increment-a-version-in-every-plugin) by incrementing patch versions for each component.
- [x] Lower the [frequency of builds](https://github.com/opensearch-project/opensearch-build/pull/1475) for this version of OpenSearch and/or OpenSearch Dashboards.
- [x] Update [this template](https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md) with any new or missed steps.
- [ ] Create an issue for a retrospective, solicit feedback, and publish a summary.
- - [x] https://github.com/opensearch-project/opensearch-build/issues/1943

### Components

__Replace with links to all component tracking issues.__

| Component | On track | Notes |
| --------- | -------- | ----- |
| {COMPONENT_ISSUE_LINK} | {INDICATOR}} | {STATUS} |

<details><summary>Legend</summary>
<p>

| Symbol | Meaning |
| -------- | ---------- |
| :green_circle: | On track with overall release |
| :yellow_circle: | Missed last milestone |
| :red_circle: | Missed multiple milestones |

</p>
</details>

**Comment by downsrob** [comment]
@abhinavGupta16 In previous releases we didn't need to bump dashboards plugins versions when the plugins didn't have any changes. Have we regressed in this release?

**Comment by abhinavGupta16** [comment]
> @abhinavGupta16 In previous releases we didn't need to bump dashboards plugins versions when the plugins didn't have any changes. Have we regressed in this release?

@downsrob - We always have to bump the plugins if the core is bumped, for dashboards too

**Comment by kavilla** [comment]
x64
```
| functionalTestDashboards | with-security        | PASS  |    0 |
| functionalTestDashboards | without-security     | FAIL  |    3 |
```

arm64
```
| functionalTestDashboards | with-security        | FAIL  |    4 |
| functionalTestDashboards | without-security     | FAIL  |    5 |
```

**Comment by kavilla** [comment]
x64 failures without security
```` 
     Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  core-opensearch-dashboards/opensear      00:40       34       34        -        -        - │
  │    ch-dashboards/dashboard_sanity_test                                                         │
  │    _spec.js                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  plugins/alerting-dashboards-plugin/      02:39        3        1        2        -        - │
  │    acknowledge_alerts_modal_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      02:19        5        5        -        -        - │
  │    alert_spec.js                                                                               │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      01:22        2        2        -        -        - │
  │    alerts_dashboard_flyout_spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      00:45        4        4        -        -        - │
  │    bucket_level_monitor_spec.js                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      00:16        4        4        -        -        - │
  │    destination_spec.js                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      00:18        4        4        -        -        - │
  │    query_level_monitor_spec.js                                                                 │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:15        1        1        -        -        - │
  │    s-plugin/create_detector_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:10        8        8        -        -        - │
  │    s-plugin/dashboard_spec.js                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:08        5        5        -        -        - │
  │    s-plugin/detector_configuration_spe                                                         │
  │    c.js                                                                                        │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:14       11       11        -        -        - │
  │    s-plugin/detector_list_spec.js                                                              │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      01:05        6        6        -        -        - │
  │    s-plugin/historical_analysis_spec.j                                                         │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:10        6        6        -        -        - │
  │    s-plugin/overview_spec.js                                                                   │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:10        3        3        -        -        - │
  │    s-plugin/real_time_results_spec.js                                                          │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:19        3        3        -        -        - │
  │    s-plugin/sample_detector_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/gantt-chart-dashboards/gant      00:33        9        9        -        -        - │
  │    t_ui.spec.js                                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  plugins/index-management-dashboards      01:19        3        2        1        -        - │
  │    -plugin/indices_spec.js                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      01:06        5        5        -        -        - │
  │    -plugin/managed_indices_spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:24        5        5        -        -        - │
  │    -plugin/policies_spec.js                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:41        4        4        -        -        - │
  │    -plugin/rollups_spec.js                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:32        4        4        -        -        - │
  │    -plugin/transforms_spec.js                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/ap      02:40       20       16        -        4        - │
  │    p_analytics.spec.js                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/ev      01:45       24       22        -        2        - │
  │    ent_analytics.spec.js                                                                       │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/no      04:12       28       26        -        2        - │
  │    tebooks.spec.js                                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/pa      07:13       26       26        -        -        - │
  │    nels.spec.js                                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:41        8        4        -        4        - │
  │    ace_analytics_dashboard.spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:35        6        5        -        1        - │
  │    ace_analytics_services.spec.js                                                              │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:30        5        5        -        -        - │
  │    ace_analytics_traces.spec.js                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/query-workbench-dashboards/      01:36       20       20        -        -        - │
  │    ui.spec.js                                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/01-creat      04:09       14       14        -        -        - │
  │    e.spec.js                                                                                   │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/02-edit.      01:21        3        3        -        -        - │
  │    spec.js                                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/03-detai      01:57        6        6        -        -        - │
  │    ls.spec.js                                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/04-downl      01:16        5        5        -        -        - │
  │    oad.spec.js                                                                                 │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/audit_log_spec.js         1ms        -        -        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/auth_spec.js              1ms        -        -        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/get_started_spec.j        1ms        -        -        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/internalusers_spec        1ms        -        -        -        -        - │
  │    .js                                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/permissions_spec.j        2ms        -        -        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/roles_spec.js             2ms        -        -        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/tenants_spec.js           1ms        -        -        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✖  2 of 40 failed (5%)                      43:32      294      278        3       13        -  
````

**Comment by kavilla** [comment]
arm64 failures with security

````
       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  core-opensearch-dashboards/opensear      00:37       34       34        -        -        - │
  │    ch-dashboards/dashboard_sanity_test                                                         │
  │    _spec.js                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      01:19        3        3        -        -        - │
  │    acknowledge_alerts_modal_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      02:18        5        5        -        -        - │
  │    alert_spec.js                                                                               │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      01:24        2        2        -        -        - │
  │    alerts_dashboard_flyout_spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  plugins/alerting-dashboards-plugin/      01:45        4        3        1        -        - │
  │    bucket_level_monitor_spec.js                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      00:18        4        4        -        -        - │
  │    destination_spec.js                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      00:22        4        4        -        -        - │
  │    query_level_monitor_spec.js                                                                 │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:16        1        1        -        -        - │
  │    s-plugin/create_detector_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:16        8        8        -        -        - │
  │    s-plugin/dashboard_spec.js                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:15        5        5        -        -        - │
  │    s-plugin/detector_configuration_spe                                                         │
  │    c.js                                                                                        │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:21       11       11        -        -        - │
  │    s-plugin/detector_list_spec.js                                                              │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      01:11        6        6        -        -        - │
  │    s-plugin/historical_analysis_spec.j                                                         │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:14        6        6        -        -        - │
  │    s-plugin/overview_spec.js                                                                   │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:12        3        3        -        -        - │
  │    s-plugin/real_time_results_spec.js                                                          │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:23        3        3        -        -        - │
  │    s-plugin/sample_detector_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/gantt-chart-dashboards/gant      00:40        9        9        -        -        - │
  │    t_ui.spec.js                                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:22        3        3        -        -        - │
  │    -plugin/indices_spec.js                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      01:07        5        5        -        -        - │
  │    -plugin/managed_indices_spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:27        5        5        -        -        - │
  │    -plugin/policies_spec.js                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:44        4        4        -        -        - │
  │    -plugin/rollups_spec.js                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:27        4        4        -        -        - │
  │    -plugin/transforms_spec.js                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  plugins/observability-dashboards/ap      05:45       20       13        3        4        - │
  │    p_analytics.spec.js                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/ev      02:01       24       22        -        2        - │
  │    ent_analytics.spec.js                                                                       │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/no      04:57       28       26        -        2        - │
  │    tebooks.spec.js                                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/pa      07:29       26       26        -        -        - │
  │    nels.spec.js                                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:52        8        4        -        4        - │
  │    ace_analytics_dashboard.spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:39        6        5        -        1        - │
  │    ace_analytics_services.spec.js                                                              │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:34        5        5        -        -        - │
  │    ace_analytics_traces.spec.js                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/query-workbench-dashboards/      01:30       20       20        -        -        - │
  │    ui.spec.js                                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/01-creat      04:17       14       14        -        -        - │
  │    e.spec.js                                                                                   │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/02-edit.      01:22        3        3        -        -        - │
  │    spec.js                                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/03-detai      01:59        6        6        -        -        - │
  │    ls.spec.js                                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/04-downl      01:17        5        5        -        -        - │
  │    oad.spec.js                                                                                 │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/audit_log_spec.js       00:07        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/auth_spec.js            00:04        2        2        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/get_started_spec.j      00:10        8        8        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/internalusers_spec      00:08        4        4        -        -        - │
  │    .js                                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/permissions_spec.j      00:11        5        5        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/roles_spec.js           00:07        4        4        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/tenants_spec.js         00:05        2        2        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✖  2 of 40 failed (5%)                      48:51      323      306        4       13        -  
```

**Comment by kavilla** [comment]
arm64 without security

```
       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  core-opensearch-dashboards/opensear      00:34       34       34        -        -        - │
  │    ch-dashboards/dashboard_sanity_test                                                         │
  │    _spec.js                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  plugins/alerting-dashboards-plugin/      02:41        3        1        2        -        - │
  │    acknowledge_alerts_modal_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      02:18        5        5        -        -        - │
  │    alert_spec.js                                                                               │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      01:25        2        2        -        -        - │
  │    alerts_dashboard_flyout_spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      00:57        4        4        -        -        - │
  │    bucket_level_monitor_spec.js                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      00:15        4        4        -        -        - │
  │    destination_spec.js                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/alerting-dashboards-plugin/      00:20        4        4        -        -        - │
  │    query_level_monitor_spec.js                                                                 │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:16        1        1        -        -        - │
  │    s-plugin/create_detector_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:12        8        8        -        -        - │
  │    s-plugin/dashboard_spec.js                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:10        5        5        -        -        - │
  │    s-plugin/detector_configuration_spe                                                         │
  │    c.js                                                                                        │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:18       11       11        -        -        - │
  │    s-plugin/detector_list_spec.js                                                              │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      01:08        6        6        -        -        - │
  │    s-plugin/historical_analysis_spec.j                                                         │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:11        6        6        -        -        - │
  │    s-plugin/overview_spec.js                                                                   │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:09        3        3        -        -        - │
  │    s-plugin/real_time_results_spec.js                                                          │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/anomaly-detection-dashboard      00:18        3        3        -        -        - │
  │    s-plugin/sample_detector_spec.js                                                            │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/gantt-chart-dashboards/gant      00:35        9        9        -        -        - │
  │    t_ui.spec.js                                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  plugins/index-management-dashboards      01:19        3        2        1        -        - │
  │    -plugin/indices_spec.js                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      01:05        5        5        -        -        - │
  │    -plugin/managed_indices_spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:22        5        5        -        -        - │
  │    -plugin/policies_spec.js                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:38        4        4        -        -        - │
  │    -plugin/rollups_spec.js                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/index-management-dashboards      00:25        4        4        -        -        - │
  │    -plugin/transforms_spec.js                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✖  plugins/observability-dashboards/ap      04:49       20       14        2        4        - │
  │    p_analytics.spec.js                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/ev      01:59       24       22        -        2        - │
  │    ent_analytics.spec.js                                                                       │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/no      04:52       28       26        -        2        - │
  │    tebooks.spec.js                                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/pa      07:30       26       26        -        -        - │
  │    nels.spec.js                                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:52        8        4        -        4        - │
  │    ace_analytics_dashboard.spec.js                                                             │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:37        6        5        -        1        - │
  │    ace_analytics_services.spec.js                                                              │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/observability-dashboards/tr      00:33        5        5        -        -        - │
  │    ace_analytics_traces.spec.js                                                                │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/query-workbench-dashboards/      01:29       20       20        -        -        - │
  │    ui.spec.js                                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/01-creat      04:17       14       14        -        -        - │
  │    e.spec.js                                                                                   │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/02-edit.      01:22        3        3        -        -        - │
  │    spec.js                                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/03-detai      01:59        6        6        -        -        - │
  │    ls.spec.js                                                                                  │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/reports-dashboards/04-downl      01:17        5        5        -        -        - │
  │    oad.spec.js                                                                                 │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/audit_log_spec.js         2ms        -        -        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/auth_spec.js              2ms        -        -        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/get_started_spec.j        2ms        -        -        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/internalusers_spec        1ms        -        -        -        -        - │
  │    .js                                                                                         │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/permissions_spec.j        1ms        -        -        -        -        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/roles_spec.js             2ms        -        -        -        -        - │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  plugins/security/tenants_spec.js           1ms        -        -        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✖  3 of 40 failed (8%)                      47:30      294      276        5       13        -  
```

**Comment by bbarani** [comment]
Closing this issue as we released 1.3.1 OpenSearch Dashboards.
